guides/tutorial/start-tutorial.md

```diff
@@ -444,5 +444,5 @@ Now exit iex and start up your server:
 $ mix phx.server
 ```
 
-Enter `localhost:4000/protected` in your browser's address bar, and you should see "unathenticated".  Now, enter `localhost:4000/login` in your browser's address bar, and login with your `me` username and `secret` password.  You should automatically be redirected to the protected page, which you can now see!  To logout, enter `localhost:4000/logout` in your browser's address bar, and you will be redirected to the login page.  Instead of logging in, enter `localhost:4000/protected` in your browser's address bar, and you will see "unathenticated" again!
+Enter `localhost:4000/protected` in your browser's address bar, and you should see "unauthenticated".  Now, enter `localhost:4000/login` in your browser's address bar, and login with your `me` username and `secret` password.  You should automatically be redirected to the protected page, which you can now see!  To logout, enter `localhost:4000/logout` in your browser's address bar, and you will be redirected to the login page.  Instead of logging in, enter `localhost:4000/protected` in your browser's address bar, and you will see "unauthenticated" again!
```

Comment:
Is this line more than 80 characters width? Would you mind to break it down into multiple columns? It is really hard to figure what exactly changes.

Reply:
there was just a typo in "unauthenticated" that I fixed! :)